Scripts/Swift/SetUpiOSProject.swift

```diff
@@ -0,0 +1,184 @@
+            case 5: interface = .init(argument)
+            default: break
+            }
+        }
+
+        if isCI {
+            minimumVersion = "14.0"
+        }
+
+        while bundleIdProduction.isEmpty || !checkPackageName(bundleIdProduction) {
+            print("BUNDLE ID PRODUCTION (i.e. com.example.project):")
+            bundleIdProduction = readLine() ?? ""
+        }
+        while bundleIdStaging.isEmpty || !checkPackageName(bundleIdStaging)  {
+            print("BUNDLE ID STAGING (i.e. com.example.project.staging):")
+            bundleIdStaging = readLine() ?? ""
+        }
+        while projectName.isEmpty {
+            print("PROJECT NAME (i.e. NewProject):")
+            projectName = readLine() ?? ""
+        }
+        while minimumVersion.isEmpty || !checkVersion(minimumVersion) {
+            print("iOS Minimum Version (i.e. 14.0):")
+            minimumVersion = readLine() ?? "14.0"
+        }
+        while interface == nil {
+            print("Interface [(S)wiftUI or (U)IKit]:")
+            interface = SetUpInterface.Interface(readLine() ?? "")
+        }
+        let projectNameNoSpace = projectName.trimmingCharacters(in: .whitespacesAndNewlines)
+
+        print("=> 🐢 Starting init \(projectName) ...")
+
+        print("=> 🔎 Replacing files structure...")
+
+        let fileManager = FileManager.default
+
+        // Rename test folder structure
+        fileManager.rename(file: "\(CONSTANT_PROJECT_NAME)Tests", to: "\(projectNameNoSpace)Tests")
+
+        // Rename KIF UI Test folder structure
+        fileManager.rename(file: "\(CONSTANT_PROJECT_NAME)KIFUITests", to: "\(projectNameNoSpace)KIFUITests")
+
+        // Rename app folder structure
+        fileManager.rename(file: "\(CONSTANT_PROJECT_NAME)", to: "\(projectNameNoSpace)")
+
+        // Duplicate the env example file to env file
+        fileManager.copy(file: ".env.example", to: ".env")
+
+        // Add AutoMockable.generated.swift file
+        fileManager.createFile(name: "AutoMockable.generated.swift", at: "\(projectNameNoSpace)Tests/Sources/Mocks/Sourcery")
+
+        // Add AutoMockable.generated.swift file
+        fileManager.createFile(name: "R.generated.swift", at: "\(projectNameNoSpace)/Sources/Supports/Helpers/Rswift")
+
+        // Select the Interface
+        SetUpInterface().perform(interface ?? .uiKit, projectName)
+
+        print("✅  Completed")
+
+        // Search and replace in files
+
+        print("=> 🔎 Replacing package and package name within files...")
+
+        try fileManager.replaceAllOccurrences(of: CONSTANT_BUNDLE_STAGING, to: bundleIdStaging)
+        try fileManager.replaceAllOccurrences(of: CONSTANT_BUNDLE_PRODUCTION, to: bundleIdProduction)
+        try fileManager.replaceAllOccurrences(of: CONSTANT_PROJECT_NAME, to: projectNameNoSpace)
+        try fileManager.replaceAllOccurrences(of: CONSTANT_MINIMUM_VERSION, to: minimumVersion)
+        print("✅  Completed")
```

Comment:
Regarding the project info replacement, such as project name, package, app id, etc., we might consider using a [Mustache](https://mustache.github.io/mustache.5.html) template implementation in Swift https://github.com/groue/GRMustache.swift to reduce the boilerplate code and even make the core template to be more flexible with extra mustache supported rules 🤔 

Just my suggestion for the next steps, not in this PR @nimblehq/ios-chapter. This is what we're doing in Flutter https://github.com/nimblehq/flutter-templates/tree/develop/bricks/template/__brick__/%7B%7Bproject_name.snakeCase()%7D%7D 😛

Reply:
@luongvo What do you use to generate folders and replace in files? The [GRMustache.swift](https://github.com/groue/GRMustache.swift) seems to only generate text based on the template.

Reply:
@blyscuit In Dart, we have [mason](https://github.com/felangel/mason) tool that wraps all the necessary actions. In Kotlin or Swift, I can not find one yet, so basically, we need to keep the current logic to list files for processing, but use Mustache to process instead of our manual `replace` API in Kotlin or Swift.